bonito/model.py

```diff
@@ -33,6 +35,12 @@ class Model(Module):
     """
     def __init__(self, config):
         super(Model, self).__init__()
+        if 'qscores' not in config:
+            self.qbias = 0.0
+            self.qscale = 1.0
+        else:
+            self.qbias = config['qscores']['bias']
+            self.qscale = config['qscores']['scale']
```

Comment:
@TimD1 @kishwarshafin I thought it made sense to have the `qscale` and `qbias` in the model config.